bookstore/.stickler.yml

```diff
@@ -0,0 +1,8 @@
+linters:
+  flake8:
+  eslint:
+    config: './eslint.config'
+  stylelint:
+files:
+  ignore:
+    - '.eslintrc.json'
```

Comment:
This file should look like the one in the [Microverse Linters Repo](https://github.com/microverseinc/linters-config/blob/master/react-redux/.stickler.yml)
![image](https://user-images.githubusercontent.com/46115725/72757810-99d18600-3b96-11ea-8f2f-a317ae5c3992.png)